benchmark/AlcorPerformanceTestReport/README.md

```diff
@@ -0,0 +1,718 @@
+# Introduction  
```

Comment:
@yanmo96 Please put TOC (Table of Content) and their link to the section in the beginning of the report.